apps/nrm/views/content/does-pv-need-support.md

```diff
@@ -1,4 +1,4 @@
-<p>{{#t}}pages.does-pv-need-support.paragraph-1{{/t}} <a href="/support-organisations">{{#t}}pages.does-pv-need-support.link-text-1{{/t}}</a> {{#t}}pages.does-pv-need-support.link-text-2{{/t}}</p>
+<p>{{#t}}pages.does-pv-need-support.paragraph-1{{/t}} <a class="govuk-link" href="/support-organisations">{{#t}}pages.does-pv-need-support.link-text-1{{/t}}</a> {{#t}}pages.does-pv-need-support.link-text-2{{/t}}</p>
```

Comment:
Would it be more readable to use md syntax in this file and just extend any necessary govuk classes via the css?

Reply:
Changed the file to use md syntax instead and tested.